doc/reference/stability.rst

```diff
@@ -120,9 +120,9 @@ The table below lists the available documented APIs in Zephyr, their
      - 2.2
 
    * - :ref:`hwinfo_api`
-     - Unstable
-     - 1.14
+     - Stable
      - 1.14
+     - 2.3
```

Comment:
Will the API itself change in 2.3? Or it will just become stable? If the latter then we leave 1.14 here